ui/app/styles/components/page-header-old.scss

```diff
@@ -26,7 +26,7 @@
   }
 
   .title {
-    margin-top: $spacing-36;
+    margin-top: $spacing-16;
```

Comment:
Until we replace this component, reducing to match the `gap: 16` that the HDS page header uses